src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -457,7 +457,7 @@ enum SMART_ACTION
     SMART_ACTION_EVADE                              = 24,     // No Params
     SMART_ACTION_FLEE_FOR_ASSIST                    = 25,     // With Emote
     SMART_ACTION_CALL_GROUPEVENTHAPPENS             = 26,     // QuestID
-    SMART_ACTION_COMBAT_STOP                        = 27,     // 
+    SMART_ACTION_COMBAT_STOP                        = 27,     //
```

Comment:
should mention about the new param

Reply:
I didnt do that, my VSC just remove the blank space

Reply:
yes, that is fine. What I meant is that we should also add a comment about the new parameter

> enter a range as first param

Reply:
@P-Kito I meant in `SMART_ACTION_SET_IN_COMBAT_WITH_ZONE`, that's where you added a new param, right?

Reply:
resolved :p